community/values/src/main/java/org/neo4j/values/storable/TemporalValue.java

```diff
@@ -402,6 +461,26 @@ protected final ZoneId timezone()
         millisecond( ChronoField.MILLI_OF_SECOND, 0 ),
         microsecond( ChronoField.MICRO_OF_SECOND, 0 ),
         nanosecond( ChronoField.NANO_OF_SECOND, 0 ),
+        // Read only accessors (not assignable)
```

Comment:
This comment hints that all following enum values are read-only, while in fact only the next two are.